recipes/conda-ecosystem-site-packages-isolation/meta.yaml

```
@@ -0,0 +1,28 @@
+package:
+  name: conda-ecosystem-site-packages-isolation
+  version: 0.1.0
+
+build:
+  number: 0
+
```

Constrain any Python installations with

```suggestion

requirements:
  run_constrained:
    - python >=3.3

```